nautobot/ui/src/components/Retrieve/RenderTable.js

```diff
@@ -11,12 +10,8 @@ function RenderTable({ fields, schema, data }) {
                     const fieldSchema = schema[fieldName];
                     const fieldData = data[fieldName];
                     let url = null;
-                    if (fieldSchema?.appLabel) {
-                        url = buildUrl(
-                            fieldSchema.appLabel,
-                            fieldSchema.modelName,
-                            fieldData?.id
-                        );
+                    if (fieldSchema?.modelUrl && fieldData?.id) {
+                        url = fieldSchema?.modelUrl + fieldData?.id + "/";
```

Comment:
```suggestion
                        url = fieldSchema.modelUrl + fieldData.id + "/";
```
No need for the `?` since we are sure there would be a value at this point.